backend/src/apiserver/server/pipeline_server.go

```diff
@@ -129,7 +129,10 @@ func (s *PipelineServer) CreatePipeline(ctx context.Context, request *api.Create
 	resourceAttributes := &authorizationv1.ResourceAttributes{
 		Namespace: namespace,
 		Verb:      common.RbacResourceVerbCreate,
+		Group:     common.RbacPipelinesGroup,
+		Version:   common.RbacPipelinesVersion,
 		Resource:  common.RbacResourceTypePipelines,
+		Name:      pipelineName,
```

Comment:
I'm a little rusty on this part after a while, how does this handle shared pipelines vs namespaced pipelines?

Refer to past discussions in
* https://github.com/kubeflow/pipelines/issues/5084#issuecomment-787625055
* https://github.com/kubeflow/pipelines/issues/4197

Reply:
I took a look at the comments from past discussions and am not sure how to test for a shared pipeline to make sure the code works as expected for both cases. Could you provide any pointers to how I can go about checking to see if it's able to handle both cases?